src/NuGet.Core/NuGet.ProjectModel/Utf8JsonReaderExtensions.cs

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text.Json;
+
+namespace NuGet.ProjectModel
+{
+    internal static class Utf8JsonReaderExtensions
+    {
+        //private static readonly char[] DelimitedStringDelimiters = new char[] { ' ', ',' };
```

Comment:
This is tricky.

I think the commented out code should be deleted. 
Right now, it's not self contained, it's making people think about a different PR. 

The idea behind splitting the PR into multiple parts is to allow them to be self contained. 

Note that the standards for feature branches are the same as non feature branches, as feature branches don't get reviewed when they get merged.

Reply:
I can just remove it now then,